media/css/firefox/family-nav.less

```diff
@@ -549,6 +549,8 @@
         background-image: none;
         box-shadow: 0 1px 3px rgba(50, 50, 50, 0.4);
         position: fixed;
+        top: 0;
```

Comment:
This fixes a bug in the family sticky nav when an element comes before the `#outerwrapper`, which was caught in this test failure: https://saucelabs.com/beta/tests/7847e6fc2eee49099e3e7b84a772fd13/watch#24